da/mock/mock.go

```diff
@@ -1,14 +1,18 @@
 package mock
```

Comment:
The mock implementation will be removed and integration test will be replaced with `local-celestia-devnet` in a follow up PR.

Reply:
Follow up in #1054